src/components/Modal/index.js

```diff
@@ -133,13 +143,15 @@ export default class Modal extends Component {
                         className={className}
                         size={size}
                     >
-                        <StyledCloseButton
-                            id="modal-close-button"
-                            icon={<CloseIcon />}
-                            title="Close"
-                            onClick={this.closeModal}
-                            ref={this.buttonRef}
-                        />
+                        {showClose && (
```

Comment:
we have an internal component, RenderIf for this cases, then instead use:
```
<RenderIf isTrue={!hideCloseButton}>
    <StyledCloseButton ... />
</RenderIf>
```

Reply:
I have added this suggestion, thank you very much